### PR TITLE
Add more sane default values, and move the logging configuration into…

### LINKED ISSUE #67: Figure out how to configure the log rotation.

### PR DESCRIPTION
**Pull request by schmidtw**
… the default instead of built in.  This should fix #67.

### CODE REVIEW

**Review by denopink — APPROVED** [review]
lgtm!

File: cmd/xmidt-agent/config.go [hunk 1]
```diff
@@ -307,6 +324,27 @@ var defaultConfig = Config{
...
+			MessageKey:     "M",
+			StacktraceKey:  "S",
+			LineEnding:     zapcore.DefaultLineEnding,
+			EncodeLevel:    "capital",
+			EncodeTime:     "RFC3339Nano",
+			EncodeDuration: "string",
+			EncodeCaller:   "short",
+		},
+		Rotation: &sallust.Rotation{
+			MaxSize:    1 * 1024 * 1024, // 1MB max/file
```
Comment (denopink) [inline cmd/xmidt-agent/config.go]:
```suggestion
			MaxSize:    1  // 1MB max/file
```

looks like MaxSize's unites are already in megabytes https://pkg.go.dev/gopkg.in/natefinch/lumberjack.v2#section-readme
```
        // MaxSize is the maximum size in megabytes of the log file before it gets
	// rotated. It defaults to 100 megabytes.
```

**Review by denopink — APPROVED** [review]
just 1 suggested fix, otherwise lgtm! 🍻

**Comment by denopink** [comment]
looks like this pr closes #67

File: cmd/xmidt-agent/config.go (hunk 1 above)
Comment (schmidtw) [reply cmd/xmidt-agent/config.go]: Good catch, thank you.

**Pull request merged**